### PR TITLE
[DUOS-3030] Test MockitoExtension Refactor for Fleshing Out DAA

### PR DESCRIPTION
**Pull request by aarohinadkarni**
### Addresses
https://broadworkbench.atlassian.net/browse/DUOS-3030

### Summary
Updating the test classes touched by https://github.com/DataBiosphere/consent/pull/2297 to use MockitoExtension.

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by rushtong — APPROVED** [review]
👍🏽

File: src/test/java/org/broadinstitute/consent/http/resources/UserResourceTest.java [hunk 1]
```diff
@@ -85,54 +81,37 @@ public class UserResourceTest {
...
   }
 
   @Test
-  public void testGetMe() {
+  void testGetMe() {
     User user = createUserWithRole();
     when(userService.findUserByEmail(any())).thenReturn(user);
-    when(userService.findAllUserProperties(any())).thenReturn(createResearcherProperties());
-    when(libraryCardService.findLibraryCardsByUserId(any()))
-        .thenReturn(createLibraryCards());
```
Comment (fboulnois) [inline src/test/java/org/broadinstitute/consent/http/resources/UserResourceTest.java:114]: Can you briefly describe why these conditions (in this method and others) are no longer necessary to test?

**Review by fboulnois — COMMENTED** [review]
One question:

File: src/test/java/org/broadinstitute/consent/http/resources/UserResourceTest.java (hunk 1 above)
Comment (rushtong) [reply src/test/java/org/broadinstitute/consent/http/resources/UserResourceTest.java:114]: Including them causes Mockito to complain that they are unnecessary stubs. It's not that they aren't under test, it's that their return values are irrelevant to the test method.

**Review by fboulnois — APPROVED** [review]
👍

**Pull request merged**